nautobot/core/api/parsers.py

```diff
@@ -156,16 +192,17 @@ def row_elements_to_data(self, counter, row, serializer):
             elif isinstance(serializer_field, (serializers.DictField, serializers.JSONField)):
                 # We currently only store lists or dicts in JSONFields, never bare ints/strings.
                 # On the CSV write side, we only render dicts to JSON
-                if "{" in value or "[" in value:
-                    value = json.loads(value)
-                elif value:
-                    value = value.split(",")
-                    try:
-                        # We have some cases where it's a list of integers, such as in RackReservation.units
-                        value = [int(v) for v in value]
-                    except ValueError:
-                        # Guess not!
-                        pass
+                if value is not None:
+                    if "{" in value or "[" in value:
```

Comment:
```suggestion
                    if value.startswith("{") or value.startswith("["):
```

Reply:
We could even make it simpler (`startswith()` can take a tuple):

```python
                    if value.startswith(("{", "[")):
```